pyEpiabm/pyEpiabm/property/household_foi.py

```diff
@@ -99,7 +99,19 @@ def household_foi(infector, infectee, time: float):
                           * pyEpiabm.core.Parameters.instance().
                           household_transmission
                           * isolating * quarantine)
-
+        if infector.microcell.distancing_start_time is not None:
+            if infector.distancing_enhanced is True:
+                distancing = Parameters.instance().\
+                             intervention_params[
+                             'social_distancing'][
+                             'distancing_house_enhanced_susc']
+            else:
+                distancing = Parameters.instance().\
+                             intervention_params[
+                             'social_distancing'][
+                             'distancing_house_susc']
+        else:
+            distancing = 1
```

Comment:
I'm not sure if it is correct to put the scaling in 'foi'. According to CovidSim: "If so, then the susceptibility of all people residing in microcell currently undergoing social distancing is scaled, either by enhanced or non-enhanced effects (see below). Infectiousness functions are unaffected by social distancing." So should scaling be in 'household_susc'?

Reply:
I'm not sure if adding the 'distancing' in the foi actually influences the infectiousness. The result should be the same no matter this multiplication occurs in 'household_susc' or 'household_foi'.

Reply:
True right now, because we only use the susceptibility in the foi. But if we end up using it for something else, and this should be in the susceptibility, then it would be good to put in there now

Reply:
I've moved distancing scaling step from foi function to susc function in household, place and spatial